.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "loader"]
 	path = external/OpenCL-ICD-Loader
-	url = https://github.com/KhronosGroup/OpenCL-ICD-Loader.git
+	url = https://github.com/StreamHPC/OpenCL-ICD-Loader.git
 [submodule "headers"]
 	path = external/OpenCL-Headers
 	url = https://github.com/KhronosGroup/OpenCL-Headers.git
 [submodule "headers-cpp"]
 	path = external/OpenCL-CLHPP
-	url = https://github.com/KhronosGroup/OpenCL-CLHPP.git
+	url = https://github.com/StreamHPC/OpenCL-CLHPP.git
```

Comment:
https://github.com/KhronosGroup/OpenCL-CLHPP/pull/262 has been merged can we change this line back?

Reply:
Same as for the comment before, should be updated now